installers/armadillo.ps1

```diff
@@ -0,0 +1,19 @@
+#winget install Microsoft.VisualStudio.2022.BuildTools Kitware.CMake Git.Git
+
+#git clone https://gitlab.com/conradsnicta/armadillo-code.git
```

Comment:
I think you forgot to uncomment this lmao

Reply:
(ˉ▽ˉ；)...
yeah... just fixed it